state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -505,6 +533,12 @@ pub mod pallet {
 		UtxoConsolidation {
 			broadcast_id: BroadcastId,
 		},
+		ChannelOpeningFeePaid {
```

Comment:
We should include who opened the channel (and therefore who paid the fee). Perhaps it makes sense to make this just a `ChannelOpened` event and include who opened it, the asset, and the action?

Reply:
I wanted to avoid duplicating too much info... this is always emitted in the context of a swap or lp channel, and both of these emit their own events. What's the benefit of adding the extra info here?